src/Java/Utilities/FullScreen.md

```diff
@@ -0,0 +1,4 @@
+Example Program to Demonstrate Full Screen Render Window.
+
+Author: Bharatesh Chakravarthi
+Affiliation: VE Lab, Chung Ang University, Seoul, South Korea
```

Comment:
I suggest you delete the author/affiliation stuff in the markdown files, it is just visual clutter and is not usually done. The proper place for this will be in your profile. As you should know git automatically tracks contributors.